src/components/Prompt/index.js

```diff
@@ -12,22 +12,19 @@ export default ({
     <div className='i'>
       {description}
     </div>
-    {action
-      ? <div className='mt3'>
-          {startsWith(action, '/') 
-            ? <Link to={action}>
-                <Button type='primary'>
-                  {actionText}
-                </Button>
-              </Link>
-            : <a href={action}>
-                <Button type='primary'>
-                  {actionText}
-                </Button>
-              </a>
-          }
-        </div>
-      : null
-    }
+    <div className='mt3'>
+      {startsWith(action, '/') 
+        ? <Link to={action}>
+            <Button type='primary'>
+              {actionText}
+            </Button>
+          </Link>
+        : <a href={action}>
+            <Button type='primary'>
+              {actionText}
+            </Button>
+          </a>
+      }
+    </div>
```

Comment:
Action is now always given so each fallback prompt has an associated action:
![image](https://cloud.githubusercontent.com/assets/5497885/23436690/b0700008-fdc8-11e6-8756-769f389a3789.png)